scripts/publish-docker-images

```diff
@@ -21,8 +21,9 @@ IMAGE_NAME="amazon/amazon-ecs-agent"
 
 AWS_PROFILE=""
 STAGE_S3_BUCKET=""
-
-source $(dirname "${0}")/publishing-common.sh
+S3_ACL_OVERRIDE=""
+DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )"
+source ${DIR}/publishing-common.sh
```

Comment:
This should be quoted to protect against paths that would otherwise be misinterpreted or split here.

```suggestion
source "${DIR}/publishing-common.sh"
```